src/Bubble.js

```diff
@@ -161,6 +161,21 @@ const styles = {
       borderTopLeftRadius: 3,
     },
   }),
+  center: StyleSheet.create({
+    container: {
+      flex: 1,
+      alignItems: 'center',
+    },
+    wrapper: {
+      borderRadius: 15,
+      backgroundColor: '#7E8B99',
+      marginHorizontal: 30,
+      minHeight: 20,
+      justifyContent: 'flex-end',
+    },
+    containerToNext: {},
+    containerToPrevious: {},
```

Comment:
Sorry, I just realized how wrong I was. HandleBubbleToNext/Previous would not throw an error even if system messages would have an user.
Unfortunately I didn't manage to test iOS today, but I will be able to do that on monday.